neuralprophet/configure.py

```diff
@@ -226,7 +226,7 @@ def set_lr_finder_args(self, dataset_size, num_batches):
         )
 
     def get_reg_delay_weight(self, e, iter_progress, reg_start_pct: float = 0.66, reg_full_pct: float = 1.0):
-        progress = (e + iter_progress) / float(self.epochs)
+        progress = (e + iter_progress) / float(self.epochs)  # type: ignore
```

Comment:
is this `# type: ignore` an artifact? @noxan

Reply:
it's intentional as there is no real way to avoid the potential `None` type for epochs as we otherwise couldn't use dataclasses (they do not differ between type annotations for the attribute and the constructor).

Reply:
added a comment on it, would be great to resolve it at some point in time - would sadly for now result in a lot of boilerplate code - read me [here](https://stackoverflow.com/questions/64670320/how-to-avoid-checking-for-none-when-setting-optional-dataclass-args-in-post-in)